src/main/java/com/couchbase/lite/replicator/PullerInternal.java

```diff
@@ -515,8 +515,6 @@ public void insertDownloads(List<RevisionInternal> downloads) {
                     }
                 }
 
-                if(rev.getBody() != null) rev.getBody().release();
-
```

Comment:
Additional Information: 
- Impact to memory usage by not calling `Body.release()` at here could be minimum.
- From functionality point of view, releasing json field makes functional deficit for Database Change Listener functionality.